www/src/components/guidelines/cards/blog.js

```diff
@@ -1,21 +1,47 @@
-import React from "react"
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { mediaQueries } from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 import { MdArrowForward } from "react-icons/md"
 
-import { Box, Flex, Heading, Link, Text } from "../system"
+import { Box } from "theme-ui"
+import { Flex, Heading, Link, Text } from "../system"
 import ImagePlaceholder from "../image-placeholder"
 import Avatar from "../avatar"
 
-const BlogCard = ({ ...props }) => (
```

Comment:
why are `props` removed  ?

Reply:
We were passing mx={6} as props here, but I only wanted mx to be 6 above the breakpoints of sm, below that it should be 0 to fix the responsive layout. Also this is very subjective, but the thought which I had while making this change was that the blog card and starter card are very specific to `/guidelines/typography` here, and we are not using them anywhere else. I don't see any reason for passing mx={6} as props to them, so I went ahead and set mx={6} explicitly above the sm breakpoints.